django_ipam/base/admin.py

```diff
@@ -158,9 +158,9 @@ def response_add(self, request, *args, **kwargs):
         """
         response = super().response_add(request, *args, **kwargs)
         if request.POST.get('_popup'):
-            return HttpResponse("""
+            return HttpResponse(F"""
```

Comment:
`F"" -> f""` 

When you've made the change, feel free to move to another issue! :smile:

Reply:
:laughing: :laughing:  okay @atb00ker